www/locales/en.json

```diff
@@ -221,7 +221,7 @@
     },
     "RAAGS": {
       "ALL_RAAGS": {
-        "TEXT": "All Raags"
+        "TEXT": "All raags"
```

Comment:
i feel like the second word being capitalized improved readability

Reply:
@tarunsingh5 veerji, yeah I reverted this change back, this was a sideeffect of another text change we did earlier but wasn't pushed.